Scripts/DCS-BIOS/lib/modules/aircraft_modules/C-130J.lua

```diff
@@ -233,19 +233,39 @@ C_130J:definePotentiometer("PLT_CC_LIGHTING_FLOOD_LIGHT_BRIGHTNESS", devices.LIG
+C_130J:definePushButton("PLT_ICS_H2_BUTTON", devices.VOLUME_MANAGER, 3018, 208, PLT_ICS_PANEL, "Pilot HF 2 Pull to Monitor")
+C_130J:definePotentiometer("PLT_ICS_PA_VOLUME", devices.VOLUME_MANAGER, 3005, 211, { 0, 1 }, PLT_ICS_PANEL, "Pilot PA Volume Knob")
+C_130J:definePushButton("PLT_ICS_PA_BUTTON", devices.VOLUME_MANAGER, 3019, 210, PLT_ICS_PANEL, "Pilot PA Pull to Monitor")
+C_130J:definePotentiometer("PLT_ICS_PA_GAIN", devices.VOLUME_MANAGER, 3006, 1353, { 0, 1 }, PLT_ICS_PANEL, "Pilot PA GAIN Knob")
+C_130J:definePotentiometer("PLT_ICS_VOX_SENSITIVITY", devices.VOLUME_MANAGER, 3007, 1354, { 0, 1 }, PLT_ICS_PANEL, "Pilot VOX Sensitivity Knob")
+C_130J:definePotentiometer("PLT_ICS_V1_VOLUME", devices.VOLUME_MANAGER, 3008, 213, { 0, 1 }, PLT_ICS_PANEL, "Pilot VHF 1 Volume Knob")
+C_130J:definePushButton("PLT_ICS_V1_BUTTON", devices.VOLUME_MANAGER, 3020, 212, PLT_ICS_PANEL, "Pilot VHF 1 Pull to Monitor")
+C_130J:definePotentiometer("PLT_ICS_V2_VOLUME", devices.VOLUME_MANAGER, 3009, 215, { 0, 1 }, PLT_ICS_PANEL, "Pilot VHF 2 Volume Knob")
+C_130J:definePushButton("PLT_ICS_V2_BUTTON", devices.VOLUME_MANAGER, 3021, 214, PLT_ICS_PANEL, "Pilot VHF 2 Pull to Monitor")
+C_130J:definePotentiometer("PLT_ICS_SATCOM_VOLUME", devices.VOLUME_MANAGER, 3010, 217, { 0, 1 }, PLT_ICS_PANEL, "Pilot SAT Volume Knob")
+C_130J:definePushButton("PLT_ICS_SATCOM_BUTTON", devices.VOLUME_MANAGER, 3022, 216, PLT_ICS_PANEL, "Pilot SAT Pull to Monitor")
+C_130J:definePotentiometer("PLT_ICS_PVT_VOLUME", devices.VOLUME_MANAGER, 3011, 219, { 0, 1 }, PLT_ICS_PANEL, "Pilot PVT Volume Knob")
+C_130J:definePushButton("PLT_ICS_PVT_BUTTON", devices.VOLUME_MANAGER, 3023, 218, PLT_ICS_PANEL, "Pilot PVT Pull to Monitor")
+C_130J:definePotentiometer("PLT_ICS_VOXHM_VOLUME", devices.VOLUME_MANAGER, 3012, 221, { 0, 1 }, PLT_ICS_PANEL, "Pilot VOX/HM Volume Knob")
+C_130J:definePushButton("PLT_ICS_VOXHM_BUTTON", devices.VOLUME_MANAGER, 3024, 220, PLT_ICS_PANEL, "Pilot VOX/HM Pull to Monitor")
+C_130J:definePotentiometer("PLT_ICS_U1_VOLUME", devices.VOLUME_MANAGER, 3013, 223, { 0, 1 }, PLT_ICS_PANEL, "Pilot UHF 1 Volume Knob")
+C_130J:definePushButton("PLT_ICS_U1_BUTTON", devices.VOLUME_MANAGER, 3025, 222, PLT_ICS_PANEL, "Pilot UHF 1 Pull to Monitor")
+C_130J:definePotentiometer("PLT_ICS_U2_VOLUME", devices.VOLUME_MANAGER, 3014, 225, { 0, 1 }, PLT_ICS_PANEL, "Pilot UHF 2 Volume Knob")
+C_130J:definePushButton("PLT_ICS_U2_BUTTON", devices.VOLUME_MANAGER, 3026, 224, PLT_ICS_PANEL, "Pilot UHF 2 Pull to Monitor")
+C_130J:defineRockerSwitch("PLT_ICS_INT_RADIO_ROCKER", devices.VOLUME_MANAGER, 3015, 3015, 3015, 3015, 291, PLT_ICS_PANEL, "Pilot Intercom/Radio Transmit")
+C_130J:defineMultipositionSwitch("PLT_ICS_INTERPHONE_MODE", devices.VOLUME_MANAGER, 3027, 293, 4, 0.3, PLT_ICS_PANEL, "Pilot Interphone Mode Switch")
+C_130J:defineTumb("PLT_ICS_TRANSMISSION_SELECTOR", devices.VOLUME_MANAGER, 3028, 294, 0.11, { -2, 7 }, nil, false, PLT_ICS_PANEL, "Pilot Transmission Selector Switch")
```

Comment:
This presently outputs 16-25 (and has an input with 82 positions), which is definitely not correct. Make sure you're testing the functionality of all switches in bort, checking both inputs and outputs.

<img width="634" height="710" alt="Image" src="https://github.com/user-attachments/assets/b2c962fd-93d6-4585-b97b-6bca6d78ecd2" />

The limits for this switch are defined in clickabledata.lua as `{(-1/9)*2, 0.8}`. To be honest I'm not sure why the last one is 0.8 - I would have expected 7/9 for a 10-position switch starting at -2/9. I went with 7/9 in the bios code since it works just as well and makes a bit more sense. I tested this and all inputs and the output seem to work without issues.
```suggestion
C_130J:defineTumb("PLT_ICS_TRANSMISSION_SELECTOR", devices.VOLUME_MANAGER, 3028, 294, 1 / 9, { -2 / 9, 7 / 9 }, nil, false, PLT_ICS_PANEL, "Pilot Transmission Selector Switch")
```